src/python/pants/engine/fs.py

```diff
@@ -76,33 +76,15 @@ def __init__(
                       with `!`, e.g. `!ignore.py`.
         :param glob_match_error_behavior: whether to warn or error upon match failures
         :param conjunction: whether all `include`s must match or only at least one must match
-        :param description_of_origin: a human-friendly description of where this PathGlobs request is
-                                      coming from, used to improve the error message for unmatched
-                                      globs. For example, this might be
-                                      "the option `--isort-config`".
```

Comment:
The quotes are intentional. The full phrase is an example value.

Reply:
The docformatter PR has left several docstrings utterly mangled, so I assumed the same was happening here. Will fix and make the intent more clear.

Reply:
Got it, sorry for having to deal with the merge conflict! Docformatter was happy with how things were previously, so you should be able to revert any changes to docstring in this PR and keep docformatter happy.

Reply:
I wasn't referring to merge conflicts, rather that newlines were removed in many docstrings even where necessary and I don't have the energy to fix them. The python 3 migration was done in steps to make it more feasible to humanly review -- I wish we had done that before assuming we knew how long it would take to migrate to black.